src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/AuctionBINWarning.java

```diff
@@ -105,13 +109,16 @@ public boolean onMouseClick(Slot slotIn, int slotId, int clickedButton, int clic
 				lowestPrice = (int) NotEnoughUpdates.INSTANCE.manager.auctionManager.getItemAvgBin(internalname);
 			}
 
-			//TODO: Add option for warning if lowest price does not exist
+			float undercutFactor = 1 - NotEnoughUpdates.INSTANCE.config.ahTweaks.warningThreshold / 100;
+			if (undercutFactor < 0) undercutFactor = 0;
+			if (undercutFactor > 1) undercutFactor = 1;
+			float overcutFactor = 1 - NotEnoughUpdates.INSTANCE.config.ahTweaks.overcutWarningThreshold / 100;
```

Comment:
Perhaps this should be `1 + ...` with the coercing adjusted accordingly? Not tested this out, but just looking at the config option description this feels wrong.

Reply:
oh while im at it: there isnt really a reason to limit the overcut percentage to 100% max, is there?